ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -649,7 +649,7 @@ remoteNetworkErrorPolicy = ErrorPolicies {
     veryShortDelay = 1 -- seconds
 
 -- | Error policy for local clients.  This is equivalent to
--- 'nullErrorPolicies', but explicit in the errors which can be catched.
+-- 'nullErrorPolicies', but explicit in the errors which can be cached.
```

Comment:
Here I think it should be _caught_ instead of _catched_, right? _cached_ is a different thing.

Reply:
done